pkgs/tools/wayland/wlxmirror/default.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, buildDotnetModule
+, fetchFromGitHub
+, wayland
+, libglvnd
+, glfw-wayland
+}:
+
+buildDotnetModule {
+  pname = "wlxmirror";
+  version = "2023-07-05";
```

```suggestion
  version = "unstable-2023-07-05";
```
or better use the new format based on the latest tag

---

@SuperSandro2000 you can't do that in buildDotnetModule, it causes it to fail, saying it's an invalid version

`> /nix/store/nldnia8a13619bmyrb3y8npcyrzxzh42-dotnet-sdk-6.0.412/sdk/6.0.412/NuGet.targets(132,5): error : 'unstable-2023-07-05' is not a valid version string. (Parameter 'value') [/build/source/WlxMirror.sln]`